setup.py

```diff
@@ -54,7 +54,7 @@ def get_packages(package):
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        "h11>=0.11,<0.13",
+        "h11>=0.11,<=0.13",
```

Comment:
```suggestion
        "h11>=0.11,<0.14",
```

It's rather unlikely that e.g. 0.13.1 will break anything.